test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -3459,9 +3483,12 @@ BOOST_AUTO_TEST_CASE(array_pop_array_transition)
 			}
 		}
 	)";
-	compileAndRun(sourceCode);
-	ABI_CHECK(callContractFunction("test()"), encodeArgs(1, 2, 3));
-	BOOST_CHECK(storageEmpty(m_contractAddress));
+	ALSO_VIA_YUL(
+		DISABLE_EWASM_TESTRUN()
+		compileAndRun(sourceCode);
+		ABI_CHECK(callContractFunction("test()"), encodeArgs(1, 2, 3));
+		BOOST_CHECK(storageEmpty(m_contractAddress));
+	);
```

Comment:
This test fails as there is still some missing feature (storage to storage copy)